engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -354,48 +464,80 @@ public static String removeComments(String s) {
      *         package.class.part.part[.*];"
      */
     @VisibleForTesting
-    public static String isValidImportString(Logger log, String importString) {
+    public Optional<GroovyImport> createImport(String importString) {
         // look for (ignoring whitespace): optional "import" optional "static" everything_else optional ".*" optional
+        // "as" optional ".*" optional
```

Comment:
this comment looks like it might have gotten formatted strangely by spotless (maybe needs `@formatter:off`/`@formatter:on`?)

Reply:
Aside from the line break, that's pretty much what it should say I think? The "as" syntax is new, was previously missing in DHE/DHC